terraform/modules/hub/static_ingress.tf

```diff
@@ -129,7 +129,7 @@ module "static_ingress_ecs_roles" {
   deployment       = "${var.deployment}"
   tools_account_id = "${var.tools_account_id}"
   service_name     = "static-ingress"
-  image_name       = "verify-static-ingress"
+  image_name       = "verify-static-ingress*"
```

Comment:
Do we need a star in the image_name?

Reply:
yes, resolving this due to sakis's comment